data_changes.md

```diff
@@ -0,0 +1,58 @@
+## Changes to Imported Data ##
+
+Generally, data will be imported as-is from historical data or newly given data
+with a few exceptions:
+
+- _IMPORT TYPE_: Broker, USASpending
+- _CHANGE_: **Creation of summary awards**
+- _DESCRIPTION_: Parent awards that do not currently exist in the database will be
+created with empty data, to provide references for transactions being loaded.
+- _FILE_: `submissions/management/commands/load_submission.py`, et. al.
+
+
+- _IMPORT TYPE_: Broker
+- _CHANGE_: **State code converted to foreign province**
+- _DESCRIPTION_: Cases where a location must be created, but the data specified has
+a state code, but is not also specified with country code of 'USA' will be converted
+to a foreign province
+- _FILE_: `submissions/management/commands/load_submission.py`
+
+
+- _IMPORT TYPE_: All (Location Data)
+- _CHANGE_: **Location Data is updated if a valid match is found**
+- _DESCRIPTION_: Cases where some location, but not all, data is specified, if a
+single unique reference (STATE CODE, CITY CODE, COUNTY CODE, STATE NAME, COUNTY NAME)
+is found, the location object is updated with that matching data
+- _FILE_: `references/models.py`
+
+
+- _IMPORT TYPE_: USA Spending, Contract; USA Spending, Financial
```

Comment:
If we do keep IMPORT TYPE, `USA Spending, Financial` should be `USA Spending, Assistance`